src/components/carousel/index.jsx

```diff
@@ -0,0 +1,174 @@
+import PropTypes from "prop-types";
+import Slider from "react-slick";
+
+const COMPONENT_CLASS_NAME = "c-carousel";
+
+// Removes any null or undefined properties for clarity
+const sanitizeSliderProps = (obj) => {
+	let tempProps = { ...obj };
+	tempProps = Object.entries(tempProps)
+		.filter(([, v]) => v != null)
+		.reduce((acc, [k, v]) => ({ ...acc, [k]: v }), {});
+	tempProps.className = tempProps.sliderClassName;
+	delete tempProps.sliderClassName;
+	return tempProps;
+};
+
+const Carousel = ({ children, className, ...slick }) => {
+	const sliderProps = sanitizeSliderProps(slick);
+	return (
+		<div className={className ? `${COMPONENT_CLASS_NAME} ${className}` : `${COMPONENT_CLASS_NAME}`}>
+			<Slider {...sliderProps}>{children}</Slider>
+		</div>
+	);
+};
+
+Carousel.defaultProps = {
+	accessibility: true,
```

Comment:
We shouldn't be defining all the props from the underly library within our component - we could use a `sliderSettings` object that passes through to the slider like so ```jsx
<Slider {...sliderSettings} />```

Reply:
You don't need to define the underlying library props though, we can just document that the `slideSettings` object has the ability to set any of the {library name} options to it

Reply:
@ecarlisle - You asked the same question twice? Was that a mistake?

Reply:
LOL, yes. Sorry, copy and paste & multiple tabs. :-D

Reply:
@matthewroach, @ecarlisle regarding having the react slick props in its own prop object. I like the clarity, but couple issues that I am having with it. 1) The storybook doc generator dumps out that nasty json instead a nice listing of the objects properties. The second issue is that React does not populate the default values that are defined in the object cause default props are only shallowly merged.